### PR TITLE
Query Multiple Tables

### PR DESCRIPTION
**Pull request by HafizMassamTabraizKhan**
In this PR I did the following:

- Added additional tables.
- Added join queries.

Q1
Q2
Q3
Q4
Q5
Q6
Q7

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,4 +10,29 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals 
-	ADD species TEXT;
\ No newline at end of file
+	ADD species TEXT;
+
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    full_name TEXT,
```
Comment (t-yanick) [inline schema.sql]: - [x] Great job so far. As per project requirement, the data type to use here is a string. `varchar` is most preferable string for storing names. `TEXT` will occupy more space of which it is for storing just name.
**Example**
` name varchar(255),`

File: schema.sql [hunk 2]
```diff
@@ -10,4 +10,29 @@ CREATE TABLE animals (
...
+    age INT,
+    primary key(id)
+);
+
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name TEXT,
```
Comment (t-yanick) [inline schema.sql]: - [x] Great job so far. As per project requirement, the data type to use here is a string. `varchar` is most preferable string for storing names. `TEXT` will occupy more space of which it is for storing just name.
**Example**
` name varchar(255),`

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @HafizMassamTabraizKhan , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  SQL commands put in the correct file
:heavy_check_mark: Query results screenshots added to PR body
:heavy_check_mark: Use of correct Gitflow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @HafizMassamTabraizKhan, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done implementing the changes requested by the previous reviewer. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (HafizMassamTabraizKhan) [reply schema.sql]: @t-yanick 

Hey, I want to clarify something regarding the suggested changes. The project requirements specifically ask for the initialization of a column with a string, and that's precisely what I did by selecting TEXT for this purpose. TEXT, along with CHAR() and VARCHAR(), falls under the optional suggestions for initializing a column with a string. Therefore, kindly consider this information and avoid halting the project approval based on changes that were already in line with the requirement section.

Thank you for understanding.

**Pull request merged**